source/supported-databases.txt

```diff
@@ -0,0 +1,61 @@
+.. _supported-databases:
+
+=======================================
+Supported Source Databases and Versions
+=======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Relational Migrator supports migration from source databases to MongoDB. 
+
+Definition
+----------
+
+To create a Relational Migrator project, you can connect to a source database to migrate
+to MongoDB. This page provides the supported source databases. 
+
+.. note::
+
+   It may be possible to use other versions or deployments but they have not been tested 
```

Comment:
[suggestion][ Active voice ](https://www.mongodb.com/docs/meta/style-guide/writing/use-active-voice/)(making user the subject) 

```suggestion
   You might be able to use other versions or deployments for your source database, but they have not been tested 
```

Reply:
Done.